### ISSUE TITLE
change the interface for handler with light-hybrid-4j

### ISSUE THREAD

**Issue by stevehu** [networknt/light-codegen#124]
https://github.com/networknt/light-hybrid-4j/issues/25

**Comment by stevehu** [comment]
@NicholasAzar I have made some change in light-hybrid-4j to simply add HttpServerExchange exchange to the handle interface method and updated the codegen-web accordingly. Could you please review the changes? Upon your confirmation, I will update other repositories that use light-hybrid-4j. Thanks.

**Comment by NicholasAzar** [comment]
@stevehu This looks great to me! Let me know if I can help in any of the conversions.

As part of the conversion, will you be removing the response header modifications done by the light-hybrid rpc-handlers and push them to the client handlers? (in the cases where the request actually makes it to the client handlers, and not blocked by a validation or request issue)

**Comment by stevehu** [comment]
I think that is a good idea. However, let's make the signature changes first and we can tackle it in another issue. I will update the examples and light-portal with the light-bot build as the problem will be easily identified by the build process. Thanks a lot for the review.
